src/Lumper.UI/Views/MainWindow.axaml

```diff
@@ -33,12 +33,14 @@
             <MenuItem Header="_File">
               <MenuItem Header="_Open..." Command="{Binding OpenCommand}" HotKey="Ctrl+O"
                         InputGesture="Ctrl+O" />
+              <MenuItem Header="_Open from URL..." Command="{Binding OpenUrlCommand}"
+                        HotKey="Ctrl+Shift+O"
+                        InputGesture="Ctrl+Shift+O" />
               <Separator />
-
               <MenuItem Header="_Save" Command="{Binding SaveCommand}" HotKey="Ctrl+S"
                         InputGesture="Ctrl+S" />
-              <MenuItem Header="_Save As..." Command="{Binding SaveAsCommand}" HotKey="Ctrl+Alt+S"
-                        InputGesture="Ctrl+Alt+S" />
+              <MenuItem Header="_Save As..." Command="{Binding SaveAsCommand}" HotKey="Ctrl+Shift+S"
+                        InputGesture="Ctrl+Shift+S" />

```

Comment:
??

Reply:
Still dunno what this means

Reply:
You've changed the hotkey from Alt to Shift because "loading BSPs over HTTP" and thats not true so it should be in a different commit .. but doesn't matter at this point